plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-methods-block/style.scss

```diff
@@ -11,4 +11,11 @@
 	.wc-block-components-shipping-rates-control__no-results-notice {
 		margin: em($gap-small) 0;
 	}
+
+	.wc-block-components-shipping-rates-control__no-shipping-address-message {
+		background-color: $gray-200;
```

Comment:
I chose the combination of grays that was closest to the design while also providing enough contrast by default on TT4